packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -136,6 +136,8 @@ class BottomSheet extends Component {
 				onBackdropPress={ this.props.onClose }
 				onBackButtonPress={ this.props.onClose }
 				onSwipe={ this.props.onClose }
+				onDismiss={ Platform.OS === 'ios' ? this.props.onDismiss : undefined }
```

Comment:
Think we can safely leave these props here now. They will be useful as a reference in the future in case we want to call a method once the BottomSheet is closed.